docs/en/integrations/data-ingestion/kafka/cloud/confluent/index.md

```diff
@@ -148,3 +150,11 @@ The instructions for creating an HTTP Sink in Confluent Cloud can be found [here
     * `Maximum Reties` - the default (10) is appropriate but feel to adjust for more robust retries.
 
 <img src={require('./images/http_sink_config.png').default} class="image" alt="Connecting Confluent HTTP Sink" style={{width: '50%'}}/>
+
+### Troubleshooting
+#### HTTP Sink doesn't batch messages
+
+From the [Sink documentation](https://docs.confluent.io/kafka-connectors/http/current/overview.html#http-sink-connector-for-cp):
+> The HTTP Sink connector does not batch requests for messages containing Kafka header values that are different.
+
+Verify your messages have the same header.
```

Comment:
I would change the wording batching will be activated only if kafka record has the same key

Reply:
@mzitnik I think this is now resolved, can I merge?

Reply:
@DanRoscigno feel free to merge if changes are good for you